frontend/js/tests/cb-review/CBReviewModal.test.tsx

```diff
@@ -77,7 +78,12 @@ describe("CBReviewModal", () => {
   });
 
   it("requires the user to be logged in, showing a message otherwise", async () => {
-    renderWithProviders(<NiceModal.Provider />, { critiquebrainzAuth: {} });
+    renderWithProviders(
```

Comment:
I think it might be easier to add the BrowserRouter element in the renderWithProviders helper that was created for this sort of situation

Reply:
Thanks! This is a much better solution, adding BrowserRouter in renderWithProviders helper cleaned up a good amount of code